### PR TITLE
Micu/open at #301

### PR DESCRIPTION
**Pull request by micuintus**

### CODE REVIEW

File: qml/components-felgo/TimePickerDialog.qml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import Felgo 3.0
```
Comment (jmastr) [inline qml/components-felgo/TimePickerDialog.qml:1]: Is this from `/showcase/qml/controls/TimePickerDialog.qml`?

File: translations/harbour-berlin-vegan-de.ts [hunk 2]
```diff
@@ -52,9 +52,9 @@
         <translation>omnivor</translation>
     </message>
     <message id="id-omnivorous-labeled">
-        <source>omnivorous \n(vegan dishes labeled)</source>
-        <oldsource>omnivorous 
-(vegan dishes labeled)</oldsource>
+        <source>omnivorous
```
Comment (jmastr) [inline translations/harbour-berlin-vegan-de.ts:55]: Wanna fix the incorrect indentation?

File: qml/components-felgo/TimePickerDialog.qml (hunk 1 above)
Comment (micuintus) [reply qml/components-felgo/TimePickerDialog.qml:1]: Yes, sir! See: https://github.com/micuintus/harbour-Berlin-Vegan/pull/331/commits/55ea24d686025d4cd88a5cd66b8e6a4e862a559a

File: translations/harbour-berlin-vegan-de.ts (hunk 2 above)
Comment (micuintus) [reply translations/harbour-berlin-vegan-de.ts]: Should be gone now

**Pull request merged**